lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1625,6 +1625,72 @@ class ConvertTensorStaticInfoCastOp
 };
 } // namespace
 
+namespace {
+class ConvertAtenTriuOp : public OpConversionPattern<AtenTriuOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenTriuOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    Value input = adaptor.self();
+    Value diagonal = adaptor.diagonal();
+
+    auto inputType = input.getType().cast<RankedTensorType>();
+    uint64_t inputRank = inputType.getRank();
+
+    if (inputRank < 2)
+      return op.emitError(
+          "too few dimensions to compute triangular part of matrix");
+
+    // For each dimension, compute the affine expression and the dimension size.
+    SmallVector<AffineExpr> exprs;
+    SmallVector<Value> outputDims;
+    for (auto idx = 0UL; idx < inputRank; idx++) {
+      exprs.push_back(rewriter.getAffineDimExpr(idx));
```

Comment:
`exprs` is not used

Reply:
Woops, thanks for catching that.  Removed.